Localization/de-DE/faq.yaml

```
@@ -9,5 +9,5 @@ text: [
 
 # The rest of the paragraph
 # "This section contains everything from guides and information about how to install and use the Folding@home software to the science behind our research. Look through the sidebar and find the topic you want to know more about."
-"Dieser Bereich beinhaltet alles, von Anleitungen und Informationen wie Folding@home zu installieren und benutzen ist, bis hin zu der Wissenschaft welche hinter unseren Forschungen steckt. Navigieren Sie einfach durch die Seitenleiste und suchen Sie das Thema über welches Sie mehr erfahren wollen."
+"Dieser Abschnitt enthält alles, von Anleitungen und Informationen über die Installation und Verwendung der Folding@home Software, bis hin zur Wissenschaft hinter unseren Forschung. Navigieren Sie einfach durch die Seitenleiste und finden Sie das Thema über welches Sie mehr erfahren wollen."
```

```suggestion
"Dieser Abschnitt enthält alles, von Anleitungen und Informationen über die Installation und Verwendung der Folding@home-Software, bis hin zur Wissenschaft hinter unserer Forschung. Navigieren Sie einfach durch die Seitenleiste und finden Sie das Thema über welches Sie mehr erfahren wollen."
```

---

```suggestion
"Dieser Abschnitt enthält alles, von Anleitungen und Informationen über die Installation und Verwendung der Folding@home Software, bis hin zu der Wissenschaft hinter unseren Forschung. Navigieren Sie einfach durch die Seitenleiste und finden Sie das Thema über welches Sie mehr erfahren wollen."
```
Your changes are fine for me. But sadly I was to fast with my previous acceptance. I would prefer _zu der_ before _zur_, because _zur_ does not sound that good for me.